softwerkskammer/frontendtests/fixtures/locals.js

```diff
@@ -2,16 +2,6 @@
 'use strict';
 var moment = require('moment-timezone');
 
-var addonConfig = {
-  addonInformation: function () { return 'addonInformation'; },
```

Comment:
Die hängen auch noch in den alten SoCraTes-Activities. Willst Du dafür auch ein Migrationsskript machen? (oder hast Du schon und ich habe nur noch nicht weit genug gelesen?)